Namespace has no exported member "MarkedExtension"

I was using highlight in marked.parse like so:

`
marked.parse(input, {
         highlight: function (code, lang) {
         return highlightjs.highlightAuto(code).value;
        },
        async: true,
      }) `

And because of the deprecaion warning I changed it to this:

` marked.use(
      markedHighlight({
        highlight: function (code, lang) {
          return highlightjs.highlightAuto(code).value;
        },
      }),
    );`

However when building my application I get the following error:

../node_modules/marked-highlight/src/index.d.ts(68,82): error TS2694: Namespace '"/home/myApp/node_modules/@types/marked/index"' has no exported member 'MarkedExtension'.

I already updated the typedefinitions of marked to "@types/marked": "^5.0.0",
and I tried several settings in tsconfig, but nothing helped.

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -65,7 +65,7 @@ declare module 'marked-highlight' {
    * @param options Options for the extension
    * @return A MarkedExtension to be passed to `marked.use()`
    */
-  export function markedHighlight(options: SynchronousOptions): import('marked').MarkedExtension;
+  export function markedHighlight(options: SynchronousOptions): import('marked').marked.MarkedExtension;
 
   /**
    * Configures a marked extension to apply syntax highlighing to code elements.
@@ -73,7 +73,7 @@ declare module 'marked-highlight' {
    * @param options Options for the extension
    * @return A MarkedExtension to be passed to `marked.use()`
    */
-  export function markedHighlight(options: AsynchronousOptions): import('marked').MarkedExtension;
+  export function markedHighlight(options: AsynchronousOptions): import('marked').marked.MarkedExtension;
 
   /**
    * Configures a marked extension to apply syntax highlighing to code elements.
@@ -83,5 +83,5 @@ declare module 'marked-highlight' {
    */
   export function markedHighlight(
     highlightFunction: SyncHighlightFunction
-  ): import('marked').MarkedExtension;
+  ): import('marked').marked.MarkedExtension;
 }

